src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,58 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+        Background:
+            Given a school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels       | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | [ "Tenth grade" ] | [ "School" ]                    |
+              And there is a valid AcademicWeek
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | Week One       | 2024-05-15 | 2024-05-15 | 2                      |
```

Comment:
Can we edit begin/end dates here, I think is better if they are different to avoid possible issues during search

Reply:
Done